### PR TITLE
disconnect: release resources on client disconnect

### PR DESCRIPTION
**Pull request by mmatczuk**
This PR is a followup to https://github.com/cloudwan/gohan/pull/377 protecting other (possibly) long living `gohan_` functions.

### CODE REVIEW

File: extension/otto/gohan_util.go [hunk 1]
```diff
@@ -223,14 +274,39 @@ func init() {
...
+				}
+
 				resp := map[string]string{}
 				if err != nil {
 					resp["status"] = "error"
 					resp["output"] = err.Error()
 				} else {
 					resp["status"] = "success"
-					resp["output"] = string(out)
+					resp["output"] = stdout.String()
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/otto/gohan_util.go]: This changes previous behavior. What't the cause?

Comment (mmatczuk) [inline extension/otto/gohan_util.go]: In my view they do the same thing, see implementation https://golang.org/src/bytes/buffer.go?s=1909:1941#L38. Could you elaborate a bit more what exactly has changed?

**Comment by mmatczuk** [comment]
@nati if you want me to rebase to new master just ping me.

File: extension/otto/gohan_util.go (hunk 1 above)
Comment (przemyslaw-dobrowolski-cl) [inline extension/otto/gohan_util.go]: In the previous version out buffer is read. In the changed version, stdout is read. Is out == stdout ??

Comment (przemyslaw-dobrowolski-cl) [inline extension/otto/gohan_util.go]: I've checked the code once again and it's ok.

**Pull request merged**